flutter/ARCHITECTURE.md

```diff
@@ -37,43 +34,30 @@ Table of contents
   - [`web/`](#web)
 - [Relevant Topics](#relevant-topics)
   - [Why `river_pod` and not "x" state management library?](#why-river_pod-and-not-x-state-management-library)
-  - [Why `sembast` and not "x" database?](#why-sembast-and-not-x-database)
   - [Why `mocktail` and not `mockito`?](#why-mocktail-and-not-mockito)
   - [`CoordinatorRouter` and `Router` (or Navigator 2.0)](#coordinatorrouter-and-router-or-navigator-20)
   - [Why manually importing the font, if there is the `google_fonts`?](#why-manually-importing-the-font-if-there-is-the-google_fonts)
   - [Importing Fonts](#importing-fonts)
   - [Exception handling](#exception-handling)
   - [App Icon](#app-icon)
-  - [Local collections and resources versioning](#local-collections-and-resources-versioning)
   - [Environment](#environment)
   - [Firebase](#firebase)
   - [Release](#release)
 
-## `.vscode/`
-
-While this project heavily enforces that vscode should be used, IntelliJ is also an alternative, although it won't
-provide the best experience with the setup made in this repository. If you still prefer to use it, there should be no
-problem at all, just make sure to follow the same guidelines specified in [`settings.json`](.vscode/settings.json).
-
-All configuration files exist in [`.vscode`](.vscode/) folder and **should be git-tracked**.
+# Architecture
 
-  - [`launch.json`](.vscode/launch.json) is where all pre-configured command-line scripts are at, such as running a
-  debug dev environment;
-  - [`settings.json`](.vscode/settings.json) is responsible for the editor configurations, such as line-length, rules
-  and auto-format on save.
+## `.vscode/`
 
-### Useful vscode extensions
+There is not much to add on top of the [General ARCHITECTURE.md .vscode section](//TODO), although for this Flutter
```

Comment:
Can't it be referenced from [here](https://github.com/olmps/memo/pull/194/files#diff-8f6366fd8e7a5fa30f7f05879999195b7cf5fa1e3d157822eb37f0e91d226cfdR58) ? This branch can point to that PR base branch.